database/database-cft.yaml

```diff
@@ -0,0 +1,107 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
This file includes other resources besides the database. I would recommend naming it "ga4gh-testbed-static-cft". static suggests that these resources need not be recreated while redeploying the application.

Reply:
Additionally rename `backend-cft` to "ga4gh-testbed-dynamic-cft"

Reply:
Renamed and replaced.